resources/profiles/Geeetech/process/0.08mm Extra Fine @Geeetech A30M.json

```diff
@@ -0,0 +1,13 @@
+{
+    "type": "process",
+    "name": "0.08mm Extra Fine @Geeetech A30M",
+    "inherits": "fdm_process_geeetech_0.08",
+    "from": "system",
+    "setting_id": "GP_Geeetech A30M",
+    "instantiation": "true",
+    "infill_wall_overlap": "5%",
+    "travel_speed": "100",	
+    "compatible_printers": [
+		"Geeetech A30 M 0.4 nozzle"
```

Comment:
If the parameters are exactly the same, merge the profile files. for example, add "Geeetech A30 Pro 0.4 nozzle" on compatible printers. and delete the file"0.08mm Extra Fine @Geeetech A30Pro.json".

Reply:
The profile files with the same parameters have been merged